pkg/probe/http/http_test.go

```diff
@@ -73,7 +73,7 @@ func TestHTTPProbeProxy(t *testing.T) {
 
 	go func() {
 		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-			fmt.Fprintf(w, res)
+			fmt.Fprint(w, res)
```

Comment:
This could be just `w.Write([]byte(res))`.

Reply:
Thanks! Your suggestion is good, it looks clearer.